.goreleaser.yml

```diff
@@ -0,0 +1,36 @@
+# Port flags from Makefile.
```

Comment:
We include multiple files in the release archive:
```
CHANGELOG.md
LICENSE
README.md
lokoctl
```

Perhaps we could enable option to extract directory by default, so if one just runs `tar zxvf`, files are not extracted in the work dir, but to directory?

I'm talking about:
```
    # Set to true, if you want all files in the archive to be in a single directory.
    # If set to true and you extract the archive 'goreleaser_Linux_arm64.tar.gz',
    # you get a folder 'goreleaser_Linux_arm64'.
    # If set to false, all files are extracted separately.
    # You can also set it to a custom folder name (templating is supported).
    # Default is false.
    wrap_in_directory: true
```

Reply:
Sure.